data/human/news.txt

```diff
@@ -2150,3 +2150,173 @@ news "graffiti on spaceport wall"
+			"the group "
+		word
+			"bluster at them "
+			"saber-rattle "
+		word
+			"in response "
+		word
+			", but they end up dispersing."
+			", and the two groups back down."
+
+phrase "A Light Ship"
+	word
+		"A Sparrow"
+		"An Enforcer"
+		"A Shuttle"
+		"A Heavy Shuttle"
+		"A Star Barge"
+		"A Berserker"
+		"A Fury"
+		"A Hawk"
+		"A Wasp"
+
+phrase "Pronoun"
+	word
+		"his"
+		"her"
+		"their"
+
+phrase "Careless Ends Well"
+	word
+		"turns around to reenter the ship"
+	word
+		" as it's struck by a wave, nudging it off the platform. The captain then moves the ship to a lowerable pad."
+		", moving it to a lowerable pad before it could be caught by a wave."
+
+phrase "Careless Ends Poorly"
+	word
+		"ignores air traffic control and heads to the bar. The ship"
+		"argues with the controller about their landing. The ship"
+		"turns around to reenter the ship, however it"
+	word
+		" is "
+	word
+		"struck "
+		"caught "
+	word
+		"by a wave and "
+	word
+		"swept into the sea."
+		"pushed off the pad."
+
+news "Careless Landing"
+	location
+		planet Jakobsen
+	name
+		"Recklessness"
+	message
+		phrase 
+			"A Light Ship"
+		word
+			" lands on a pad meant for heavier ships. Air traffic control is immediately upon the captain to move ${Pronoun} ship. The captain "
+		phrase
+			"Careless Ends Well"
+			"Careless Ends Poorly"
+
+phrase "CCOR Song Description"
+	word
+		"a song about "
+	word
+		"community, everyone pitching in where they can"
+		"the tyranny of a man name Bailey, until he was overthrown"
+		"CCOR, praising their accomplishments"
+		"resistance, opposition to the current planetary administration"
+
+phrase "Worker's Demands"
+	word
+		"safety regulations"
+		"increased pay"
+		"restoration of the CCOR"
+		"justice against planetary security brutality"
+		"immediate repositioning of the spaceport"
+
+news "Discontent"
+	location
+		planet Bailey
+	name
+		"Conflict"
+	message
+		word
+			"Allegations that planetary authorities will no longer move the spaceport has provoked "
+		word
+			"large scale "
+			"mass "
+			"moderate "
+			"scattered "
+		word
+			"riots "
+			"protests %"
+		word
+			"the rioters are "
+		word
+			"singing, %{CCOR Song Description}. "
+			"chanting, a list of their demands including ${Worker's Demands}. "
+			"sizing up the planetary security forces. "
+		word
+			"Security eventually steps in, "
+		word
+			"however they're pushed back. "
+			"but they're routed. "
+			"and they disperse the demonstrators. "
+			"and the demonstrators scatter but don't appear dissuaded."
+		replace
+			"% the rioters" "the protesters"
+			"%" ""
+
+news "Clandestine"
+	location
+		planet Alvorada
+name
+	"Observation"
+message
+	word
+		"You can't escape the feeling of being watched. It"
+	word
+		"'s probably the security cameras; most of them are still running."
+		" feels like there are eyes around every corner."
+		"'s the pirates; a small number of them seem to be observing you."
+	word
+		"The 
```

Comment:
```suggestion
		"After some time of this "
	word
		"a silent alarm tells you that your should check on your ship. "
		"it may be wise make sure no-one's tampering with your ship. "
	word
		"There's "
	word
		"a lone pirate prodding the ship with rudimentary tools. "
		"a few pirates watching but not apparently touching it. "
		"noone othere, the area around your ship is almost suspiciously devioid of people. "
	word
		"Whatever's going on your ship appears unscathed."

news "Microgravity"
	location
		planet Mordente-Bridi
name
	"Repairs"
message
	word
		"Watching the micro-gravity section of the spaceport from the bar, you "
	word
		"observe "
		"notice "
		"spot "
		"spy "
	word
		"a Bastion "
		"an Aerie "
		"a Falcon "
		"a Leviathan "
	word
		"returning from a raid, "
	word
		"potmarked with laser burns but no apparent serious damage. "
		"battered with thousands of small perferations. Somebody's turned a mainstay of the pirate arsenal, gatling guns, back on them. "
		"with substantial sections of hull heavily battered or missing altogether. Fair guess they didn't have enough anti-missle. "
	word
		"As it comes to a stop in the maitenance bay engineering shuttles begin "
	word
		"buzzing "
		"fluttering "
	word
		"around it, assessing the damage and beginning the work on repairs. "
	word
		"The work won't take the shuttles long."
		"They'll be working for the rest of the shift on this one."

news "Symbol War"
	location
		planet Gagarin
name
	"Guerrilla Symbology"
message
	word
		"It's an average day in a busy spaceport, "
	word
		"there's no real quiet anywhere in the port, "
		"it's the expected amout of noisy crowd background noise, "
		"with the normal buzz of activity with thousands of things happening every second, "
	word
		"it can be a bit much but anyone who works with ships learns to tune it out. "
	word
		"In spite of the drive to tune out the rumble of the crowds "
		"However, against the drive to tune out the crowds "
		"A rare and fleeting drop in the rumble allows% "
	word
		"you hear something that stands out "
	word
		"turning your attention to the unexpected noise, "
		"glancing over to the new commotion, "
	word
		"you see a squad of "
	word
		"CCOR irregulars "
		"CCOR guerrillas "
	word
		"a few of them are armed but most have paint, plaster, and ladders."
	word
		"They "
	word
		"leave the spaceport traffic alone,"
		"aren't disrupting traffic,"
	word
		"rather "
		"instead "
	word
		"they are "
	word
		"climbing the back wall toward the giant crest of the planetary government% "
		"walking along the peripheral walls of the port focusing on the small government crests "
		"returning to CCOR crests that have been covered with planetary government crests "
	word
		"they dilligently replace the government crests with CCOR's. "
	word
		"Spaceport security are "
	word
		"noticably absent."
		"actively ignoring them."
		"keeping their distance."
	replace
		"% you" " you to"
		"% they dilligently replace the government crests" " they dilligently replace the large crest"
		"%" ""

news "IP Theft"
	location
		shipyard "Betelgause Basics" "Betelgause Advanced"
		not government "Pirate"
name
	"Bootlegged Ships"
message
	word
		"A poster on the spaceport wall reads "
		"An audio broadcast playing in the spaceport recites "
		"A televised message on one of the news screens broadcasts "
	word
		"'A public-service announcement from Betelgeuse Shipyards: How do you know that a Betelgause Ship TM"
	word
		"has the time-tested quality? "
		"will live up to your high expectations? "
		"is the genuine article? "
	word
		"It's quite simple! Ensure that when buying new you do so from a reputable dealer or if you're interested in one that has itself withstood the test of time request the certificate of authenticity from the seller with your purchace. These simple precautions will ensure that "
	word
		"you're not mislead by an inferior fascimily "
		"duped with a douplicate "
		"conned with a counterfit "
	word
		"ship assembled "
	word
		"from refuse "
		"with inferior materials "
	word
		"by "
	word
		"slaves. "
		"pirates. "
		"bootleggers. "
	word
		"Buy a genuine Betelgause ship today!' The message then displays a listing of nearby planets that sell 'genuine' Betelgause ships."
```